docs/explanation/system-snaps/udisks2/index.md

```diff
@@ -0,0 +1,34 @@
+(explanation-system-snaps-udisks2-index)=
+# Udisks2
+
+The udisks snap provides a daemon and tools to access and manipulate disks and storage devices.
+
+The tools carried by the snap allows the user to:
+
+- Shows high-level and detailed information about disk drives and block devices.
+- Mount and unmounting of block devices.
+- Block device lock and unlocking.
+- Manipulating loop-devices.
+- Power off devices for safe removal.
+- SMART functionality.
+
+## What Udisks2 Offers
+
+The upstream udisks2 provides interfaces to enumerate and perform operations on disks and storage devices. Any snap with the udisks2 interface plug can access the udisksd daemon via the name `org.freedesktop.UDisks2` on the system message bus.
```

Comment:
Maybe worth adding a link to the udisks2 API spec. Also, what interface would a client snap need to connect to to use this API?

Reply:
I added some links here, and mentioned the dbus interface